cmd/bot/config.go

```diff
@@ -29,9 +31,10 @@ type config struct {
 
 	GitHub github.Config
 
-	IsCI  bool   `envconfig:"CI"`
-	PR    int    `envconfig:"DRONE_PULL_REQUEST" required:"true"`
-	Event string `envconfig:"DRONE_BUILD_EVENT"`
+	IsCI     bool   `envconfig:"CI"`
+	PR       int    `envconfig:"DRONE_PULL_REQUEST" required:"true"`
+	Event    string `envconfig:"DRONE_BUILD_EVENT"`
```

Comment:
Will remove these in a later PR as there's some local development changes that need to be made to handle.